batch_uploads_tarchive

```diff
@@ -4,46 +4,48 @@
 
 =head1 NAME
 
-batch_uploads_tarchive - upload a batch of tarchives using script tarchiveLoader
+batch_uploads_tarchive - upload a batch of DICOM archives using script
+C<tarchiveLoader>
 
 =head1 SYNOPSIS
 
 ./batch_uploads_tarchive
 
 =head1 DESCRIPTION
 
-This script uploads a list of tarchive files to the database by calling script 
+This script uploads a list of DICOM archives to the database by calling script
 C<tarchiveLoader> on each file in succession. The list of files to process is read 
-from STDIN, one file name per line. Each file name is assumed to be a path relative 
-to C<tarchiveLibraryDir> (see below).
+from C<STDIN>, one file name per line. Each file name is assumed to be a path
+relative to C<tarchiveLibraryDir> (see below).
 
 The following settings of file F<$ENV{LORIS_CONFIG}/.loris-mri/prod> affect the 
-behvavior of batch_uploads_tarchive (where C<$ENV{LORIS_CONFIG}> is the value of
-the Unix environment variable LORIS_CONFIG):
+behvaviour of batch_uploads_tarchive (where C<$ENV{LORIS_CONFIG}> is the
```

Comment:
batch_upload_tarchive should be in C<> to be consistent with your other changes

Reply:
Nice catch! Done